packages/perseus/src/article-renderer.tsx

```diff
@@ -87,21 +88,27 @@ class ArticleRenderer
         // paths, so as to check whether the focused path represents an
         // input.
         let didFocusInput = false;
+        let focusedInput;
+
         if (this._currentFocus) {
             const [sectionRef, ...focusPath] = this._currentFocus;
-            // eslint-disable-next-line react/no-string-refs
-            // @ts-expect-error - TS2339 - Property 'getInputPaths' does not exist on type 'ReactInstance'.
-            const inputPaths = this.refs[sectionRef].getInputPaths();
+
```

Comment:
String refs are old hat, and now we need a reference to the actual focused element so that we can return it to webapp with the passed in "onFocusChange" method.